web/public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow: /static/
```

Comment:
Why do we need this?

Reply:
generated file same as comments in index.html

Reply:
If there is no reason for us to keep it, I think it should be removed.